.gitignore

```diff
@@ -2,3 +2,4 @@
 npm-debug.log
 /node_modules/*
 /*.tgz
+.idea
```

Comment:
What is this for?

Reply:
InteliJ IDEs (such as WebStorm) drop config and cache files in a folder called .idea.